packages/dds/shared-object-base/api-report/shared-object-base.api.md

```diff
@@ -117,7 +117,7 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
     isAttached(): boolean;
     load(services: IChannelServices): Promise<void>;
     protected abstract loadCore(services: IChannelStorageService): Promise<void>;
-    protected readonly logger: ITelemetryLoggerExt;
+    protected readonly logger: ITelemetryBaseLogger;
```

Comment:
This one has me confused. I had to revert the broken ClassDeclaration bit in package.json because TS was complaining of an unnecessary "@ts-expects broken" line. I think that's a bug? Something having to do with the fact that `SharedObjectCore` is abstract? Consumers can extend it though, so changing the type of a protected member to something broader (which loses some of the methods of the previous type) _should_ be breaking, right?

Reply:
type test use a generice, typeonly, which will trim anything non-public